src/main/java/com/liberty/doshka/form/UserForm.java

```diff
@@ -0,0 +1,30 @@
+package com.liberty.doshka.form;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserForm {
+
+    @Id
+    @JsonProperty("id")
+    private Long userId;
+    @JsonProperty("userName")
```

Comment:
you can remove  @JsonProperty it should be used if field name and name in json are different